added various modifications (#87)
UMass-RC/unity-docs: docs/tools/ColabFold.md

> module load uri

Reviewer: does the `uri` module also require a version now?

Reviewer 2: @cissoucr This should be uri/main now.

module load MMseqs2/14-7e284-gompi-2021b
module load cudnn/cuda11-8.4.1.50
module load cuda/11.4.0

> module load miniconda

Reviewer: Add the miniconda version

Author: Good catch!

```bash
#!/bin/bash
#SBATCH --partition=gpu,gpu-preempt,gpu-long
#SBATCH --nodes=1
#SBATCH --gpus-per-node=1

> #SBATCH --mem-per-gpu=11

Reviewer: Is this accurate? Mem is referring to cpu memory, not vram as it seems above.

#### Notes:
* `<path to directory containing MSAs>` is the same as `<path to the output directory>` used with the `colabfold_search` command.
* `<path to output directory>` is the full path to an existing directory used to store the results.

> * When dealing with a large number of sequences, it is recommended to sort proteins into batches based on their size and submit a job to a GPU node with smaller VRAM for batches with shorter proteins.

Reviewer: ```suggestion
* When dealing with a large number of sequences, we recommended sorting proteins into batches based on their size and submitting a job to a GPU node with smaller VRAM for batches with shorter proteins.
```
